### ISSUE TITLE
Check failure for loading depth maps when running dense_stereo with PMVS workspace.

### ISSUE THREAD

**Issue by fabianbl** [colmap/colmap#299]
Hi @ahojnnes 

Sorry for bothering you again... I was again trying to run `dense_stereo` with a PMVS workspace, but didn't succeed so far.

My `option.txt` file looks as follows (I had more images before, but there I had the same problem):
```bash
level 1
csize 2
threshold 0.7
wsize 7
minImageNum 2
CPU 8
useVisData 1
sequence -1
timages -1 0 3
oimages 0
```

The folder structure is:
```bash
├── option.txt
├── txt
│   ├── 00000000.txt
│   ├── 00000001.txt
│   ├── 00000002.txt
├── vis.dat
├── visualize
│   ├── 00000000.jpg
│   ├── 00000001.jpg
│   ├── 00000002.jpg
```

Colmap command:
```bash
colmap dense_stereo --workspace_path hall/ --workspace_format PMVS --pmvs_option_name option.txt --DenseStereo.depth_min 0.5 --DenseStereo.depth_max 20.0
```

This successfully triggers the PatchMatch processing of the 3 views (`Processing view n / 3`), but then tries to load a depth map (`00000003.jpg.photometric.bin`) which does not exist:

```bash
...

Writing photometric output for 00000002.jpg

==============================================================================
Processing view 1 / 3
==============================================================================

Reading inputs...
F0209 09:39:42.941871  5276 mat.h:144] Check failed: text_file.is_open() hall/stereo-option.txt/depth_maps/00000003.jpg.photometric.bin
*** Check failure stack trace: ***
    @     0x7f60993c526d  google::LogMessage::Fail()
    @     0x7f60993c7083  google::LogMessage::SendToLog()
    @     0x7f60993c4dfb  google::LogMessage::Flush()
    @     0x7f60993c7a6e  google::LogMessageFatal::~LogMessageFatal()
    @           0x601c17  colmap::mvs::Mat<>::Read()
    @           0x5ffa8e  colmap::mvs::Workspace::GetDepthMap()
    @           0x607be6  colmap::mvs::PatchMatchController::ProcessProblem()
    @           0x60a9fc  std::_Function_handler<>::_M_invoke()
    @           0x60aef9  std::__future_base::_State_baseV2::_M_do_set()
    @     0x7f60986aea99  __pthread_once_slow
    @           0x60b4a2  _ZNSt17_Function_handlerIFvvEZN6colmap10ThreadPool7AddTaskIMNS1_3mvs20PatchMatchControllerEFvRKNS4_17PatchMatchOptionsEmEJPS5_S8_RmEEESt6futureINSt9result_ofIFT_DpT0_EE4typeEEOSF_DpOSG_EUlvE_E9_M_invokeERKSt9_Any_data
    @           0x7fb8ac  colmap::ThreadPool::WorkerFunc()
    @     0x7f6096ea0c80  (unknown)
    @     0x7f60986a76ba  start_thread
    @     0x7f60963e441d  clone
Aborted (core dumped)
```

Do you have an idea what could be the problem?

**Comment by ahojnnes** [comment]
Did you by any chance play around with the geom_consistency option? Or did you stop the reconstruction in the middle? For the worst, please try to clear the folder and start from scratch. Otherwise, you could share the data with me and I can have a look.

**Comment by fabianbl** [comment]
No, I didn't play around with the geom_consistency option or stop the reconstruction. I think the problem was that there is `vis.dat` file in my workspace, but I set the the `useVisData` option to false. However, this option doesn't seem to be read and the dense mapper actually [fails when `vis.dat` does not exist.](https://github.com/colmap/colmap/blob/master/src/mvs/model.cc#L318-L321)

I uploaded my sample dataset [here](https://drive.google.com/file/d/1UZeSUDrJouZu6OtDMDV7qe5NGc4zCjye/view?usp=sharing).
- Running the dense_mapper with `option-all` works.
- Using `option-subset` fails.

So, I think I will either just create a correct `vis.dat` file or convert my dataset to the colmap input format.
